projects/plugins/boost/app/rest-api/endpoints/class-update-lcp.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Save generated LCP data.
+ *
+ * This endpoint is used by WP.com to push the generated LCP data to the boost plugin.
+ */
+
+namespace Automattic\Jetpack_Boost\REST_API\Endpoints;
+
+use Automattic\Jetpack_Boost\Modules\Optimizations\Lcp\LCP_State;
+use Automattic\Jetpack_Boost\Modules\Optimizations\Lcp\LCP_Storage;
+use Automattic\Jetpack_Boost\REST_API\Contracts\Endpoint;
+use Automattic\Jetpack_Boost\REST_API\Permissions\Signed_With_Blog_Token;
+use WP_REST_Server;
+
+/**
+ * Handler for POST cloud-css/update. Expects the following body params:
+ * - success: boolean - False if the whole Critical CSS job failed.
+ * - message: string containing an error message if success is false.
+ * - providers: Object containing one result for each key:
+ *
+ * Each provider key contains:
+ * - success: boolean - False if this provider key failed.
+ * - data: Either a successful CSS block, or a CSS error.
+ *
+ * Each CSS block looks like:
+ * - css: string - containing CSS data.
+ *
+ * Each CSS error looks like:
+ * - urls: Object describing each URL which failed. Keys are URLs.
+ *
+ * Each URL failure looks like:
+ * - message: string - containing an error message.
+ * - type: string - machine readable error type.
+ * - meta: Object - JSON string compatible object containing extra metadata for consumption in the UI.
+ */
```

Comment:
This needs to be addressed before a merge. I'll take care of it.

Reply:
Addressed in [e4993c5](https://github.com/Automattic/jetpack/pull/43044/commits/e4993c558281096586a66bfe1c0018c90daf20f8) and [4a2b41f](https://github.com/Automattic/jetpack/pull/43044/commits/4a2b41f6160abf9682f8a9f92c77edf4a9f16b7e)